crt_portal/cts_forms/templates/forms/portal/header.html

```diff
@@ -11,7 +11,6 @@
           <img
             src="{% static "img/doj-logo-footer.svg" %}"
             alt=""
-            height="38"
```

Comment:
Removed this to fix the display of the header logo in IE, because I was bored in a meeting and figured I might as well 😂